packages/timeline-state-resolver/src/integrations/casparCG/__tests__/casparcg.spec.ts

```diff
@@ -2001,165 +1870,159 @@ describe('CasparCG', () => {
-		})
-		await myConductor.init()
-		await myConductor.addDevice('myCCG', {
-			type: DeviceType.CASPARCG,
-			options: {
-				host: '127.0.0.1',
-				useScheduling: false,
-				retryInterval: undefined, // disable retries explicitly, we will manually trigger them
-			},
-			commandReceiver: commandReceiver0,
-		})
-		myConductor.setTimelineAndMappings([], myLayerMapping)
-		await mockTime.advanceTimeToTicks(10000)
-
-		expect(commandReceiver0).toHaveBeenCalledTimes(0)
-
-		commandReceiver0.mockClear()
-
-		const deviceContainer = myConductor.getDevice('myCCG')
-		const device = deviceContainer!.device
-
-		// Check that no commands has been scheduled:
-		expect(await device['queue']).toHaveLength(0)
-
-		myConductor.setTimelineAndMappings([
-			{
-				id: 'video0',
-				enable: {
-					start: mockTime.getCurrentTime(), // 10000
-				},
-				layer: 'myLayer0',
-				content: {
-					deviceType: DeviceType.CASPARCG,
-					type: TimelineContentTypeCasparCg.MEDIA,
-
-					file: 'amb',
-					// transitions: {
-					// 	inTransition: {
-					// 		type: Transition.CUT,
-					// 		duration: 56 * 40
-					// 	}
-					// }
-					mixer: {
-						changeTransition: {
-							type: Transition.TSR_TRANSITION,
-							customOptions: {
-								updateInterval: 1000 / 25,
-								acceleration: 0.000002,
-							},
-						},
-						fill: {
-							x: 0,
-							y: 0,
-							xScale: 1,
-							yScale: 1,
-						},
-					},
-				},
-				keyframes: [
-					{
-						id: 'kf0',
-						enable: {
-							start: 1000, // 11000
-						},
-						content: {
-							mixer: {
-								fill: {
-									x: 0.5,
-									y: 0.5,
-									xScale: 0.5,
-									yScale: 0.5,
-								},
-							},
-						},
-					},
-				],
-			},
-		])
-
-		await mockTime.advanceTimeToTicks(10500)
-
-		// // one command has been sent:
-		expect(commandReceiver0).toHaveBeenCalledTimes(2)
-		expect(getMockCall(commandReceiver0, 0, 1)._objectParams).toMatchObject({
-			channel: 2,
-			layer: 42,
-			noClear: false,
-			clip: 'amb',
-			seek: 0,
-		})
-		expect(getMockCall(commandReceiver0, 1, 1)._objectParams).toMatchObject({
-			channel: 2,
-			layer: 42,
-			keyword: 'FILL',
-			x: 0,
-			y: 0,
-			xScale: 1,
-			yScale: 1,
-		})
-
-		commandReceiver0.mockClear()
-		await mockTime.advanceTimeToTicks(13500)
-
-		expect(
-			commandReceiver0.mock.calls.map((call) => {
-				const o = {
-					...call[1]._objectParams,
-				}
-				delete o.layer
-				delete o.channel
-				delete o.keyword
-				return o
-			})
-		).toMatchSnapshot()
-
-		expect(getMockCall(commandReceiver0, 58, 1)._objectParams).toMatchObject({
-			channel: 2,
-			layer: 42,
-			keyword: 'FILL',
-			x: 0.5,
-			y: 0.5,
-			xScale: 0.5,
-			yScale: 0.5,
-		})
-
-		expect(commandReceiver0).toHaveBeenCalledTimes(59)
-
-		commandReceiver0.mockClear()
-		await mockTime.advanceTimeToTicks(13500)
-
-		expect(commandReceiver0).toHaveBeenCalledTimes(0)
-	})
-})
+// describe('CasparCG - Custom transitions', () => {
```

Comment:
Delete commented code

Reply:
@baltedewit, what's the intention behind keeping this test? Should it be eventually uncommented?